docs/pages/smooth-scroll.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Smooth Scroll
+description: Allows internal links smooth scrolling.
+js: js/foundation.smoothScroll.js
+tags:
+  - navigation
+---
+
+<ul class="menu vertical" data-smooth-scroll>
+  <li><a href="#setup">Setup</a></li>
+  <li><a href="#javascript-reference">Javascript Reference</a></li>
+</ul>
+
+<br>
+
+## Setup
+
+To enable SmoothScroll on internal links, just add the attribute `data-smooth-scroll` to the parent container like our [Menu](menu.html). Each section needs a unique ID
+
+```html
+<ul class="horizontal menu" data-smooth-scroll>
+  <li><a href="#first">First Arrival</a></li>
+  <li><a href="#second">Second Arrival</a></li>
+  <li><a href="#third">Third Arrival</a></li>
+</ul>
+<div class="sections">
+  <section id="first">First Section</section>
+  <section id="second">Second Section</section>
+  <section id="third">Third Section</section>
+</div>
+```
+
+You can also setup SmoothScroll directly via indiviual link.
+
+```html
+<a href="#exclusive" data-smooth-scroll>Exclusive Section</a>
+<section id="exclusive">The Exclusive Section</section>
+```
```

Comment:
I agree that for a complex component, the first example must be the most common use case, because we cannot expect the user to read the entire doc and fully understand the component to know how to use it.

But after the "common use case", all features of the components must be explained gradually, not from the "simpler" to the "more complex" feature, but from the fondamental features to the more complementary ones relying on them.

However, for a so simple component with only two example, I think we can expect the whole documentation to be read, and skip the "most common case" because it would be redundant. But you can disagree with that. In any case after that, we explain the core feature: not "targeting single link" but the principle itself of smooth scrolling. Then the features relying on it: applying this behavior to a whole group of links.